app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
The `Dwarf` class should be abstract because it does not implement all abstract methods from the `Player` class. Inherit it from the `ABC` as well.